src/main/java/org/openrewrite/staticanalysis/ReplaceCollectToListWithToList.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2024 the original author or authors.
```

Comment:
```suggestion
 * Copyright 2025 the original author or authors.
```

Reply:
need recipe for this, check pipeline can do this.